server/db/index.js

```diff
@@ -1,14 +1,36 @@
 //this is the access point for all things database related!
 
-const db = require('./db')
+const db = require("./db");
 
-const User = require('./models/User')
+const User = require("./models/User");
+const Product = require("./models/Product");
+const Category = require("./models/Category");
 
-//associations could go here!
+const Order = require("./models/Order");
+
+const LineItem = require("./models/Lineitem");
+
+
+//TODO: associations could go here!
+//Product - Category = Many to Many
+
+Product.belongsToMany(Category, { through: "product_categories" });
+Category.belongsToMany(Product, { through: "product_categories" });
+
+//TODO: Lineitem  - Order should have associations
```

Comment:
Orders have many products, and products belong to many orders, _through_ the LineItem join table